modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java

```diff
@@ -134,7 +134,7 @@ public void processOpts() {
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "WebMvcConfiguration.java"));
                 supportingFiles.add(new SupportingFile("swaggerUiConfiguration.mustache",
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "SwaggerUiConfiguration.java"));
-                supportingFiles.add(new SupportingFile("application.properties",
+                supportingFiles.add(new SupportingFile("application.mustache",

```

Comment:
But you rename application.properties to swagger.properties in codegen. The target name is good ?